### PR TITLE
feat(docs): add authors analytics manual

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: docs/authors-analytics-manual.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Authors Analytics Manual
+
+Authors who have access to the publication's Google Analytics Property (News) can use the following guide to view their articles' engagement and search articles' analytics by publication language.
```
Comment (Sembauke) [inline docs/authors-analytics-manual.md:3]:
```suggestion
The following manual is for authors who have access to the Google Analytics Property (News) for their publication. It explains how to view their articles' engagement data and search for analytics by publication language.
```

File: docs/authors-analytics-manual.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+### Search  language
+
+Use the following steps in order to search the engagement reports by a publication language:
+
+![Image - Show steps to search by language on googla analytics](https://contribute.freecodecamp.org/images/google-analytics/search-by-language.png)
+
+1. Choose `News` from the top dropdown menu.
```
Comment (Sembauke) [inline docs/authors-analytics-manual.md:11]:
```suggestion
1. From the top dropdown menu, select `News`.
```

File: docs/authors-analytics-manual.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+1. Click on `Reports` from the sidebar.
```
Comment (Sembauke) [inline docs/authors-analytics-manual.md:12]:
```suggestion
1. From the sidebar, click on Reports.
```

File: docs/authors-analytics-manual.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+1. Select `Engagement` from the secondary sidebar.
```
Comment (Sembauke) [inline docs/authors-analytics-manual.md:13]:
```suggestion
1. From the secondary sidebar, select `Engagement`.
```

File: docs/authors-analytics-manual.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+1. Click on `Pages and Screens`.
+1. In the search bar, type the subpath for the desired language.
+1. Choose `Page path and screen class` from the dropdown under the search bar.
```
Comment (Sembauke) [inline docs/authors-analytics-manual.md:16]:
```suggestion
1.  From the dropdown under the search bar, choose `Page path and screen class`.
```

File: docs/authors-analytics-manual.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+# Authors Analytics Manual
+
+Authors who have access to the publication's Google Analytics Property (News) can use the following guide to view their articles' engagement and search articles' analytics by publication language.
+
+### Search  language
```
Comment (Sembauke) [inline docs/authors-analytics-manual.md:5]:
```suggestion
### Search by Language
```

File: docs/authors-analytics-manual.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+### Filter reports by Author
+
+After arriving at the `Pages and Screens` reports mentioned above, use the following steps to filter the results by specific authors.
+
+![Image - Show steps to search by language on googla analytics](https://contribute.freecodecamp.org/images/google-analytics/search-by-language.png)
+
+1. Click on the `Add filter` button.
+1. From the side navigation include `Author`.
+1. Choose an author name from the `Dimensions values` dropdown.
```
Comment (Sembauke) [inline docs/authors-analytics-manual.md:28]:
```suggestion
1. From the `Dimensions values` dropdown, choose an author's name.
```

File: docs/authors-analytics-manual.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+1. Choose `News` from the top dropdown menu.
+1. Click on `Reports` from the sidebar.
+1. Select `Engagement` from the secondary sidebar.
+1. Click on `Pages and Screens`.
+1. In the search bar, type the subpath for the desired language.
+1. Choose `Page path and screen class` from the dropdown under the search bar.
+
+
+
+### Filter reports by Author
```
Comment (Sembauke) [inline docs/authors-analytics-manual.md:20]:
```suggestion
### Filter Reports by Author
```

File: docs/authors-analytics-manual.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
+# Authors Analytics Manual
+
+Authors who have access to the publication's Google Analytics Property (News) can use the following guide to view their articles' engagement and search articles' analytics by publication language.
+
+### Search  language
+
+Use the following steps in order to search the engagement reports by a publication language:
```
Comment (Sembauke) [inline docs/authors-analytics-manual.md:7]:
```suggestion
To search for engagement reports by a specific language:
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

File: docs/authors-analytics-manual.md (hunk 1 above)
Comment (ahmadabdolsaheb) [inline docs/authors-analytics-manual.md:3]:
```suggestion
If you are an author with access to the publication's Google Analytics Property (News), you can use this guide to view your articles engagement and search for articles by publication language."
```

**Review by Sembauke — APPROVED** [review]

File: docs/authors-analytics-manual.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
+# Authors Analytics Manual
+
+If you are an author with access to the publication's Google Analytics Property (News), you can use this guide to view your articles engagement and search for articles by publication language."
```
Comment (Sboonny) [inline docs/authors-analytics-manual.md:3]:
```suggestion
If you are an author with access to the publication's Google Analytics Property (News), you can use this guide to view your articles engagement and search for articles by publication language.
```

File: docs/authors-analytics-manual.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+
+1. From the top dropdown menu, select `News`.
+1. From the sidebar, click on `Reports`.
+1. From the secondary sidebar, select `Engagement`.
+1. Click on `Pages and Screens`.
+1. In the search bar, type the subpath for the desired language.
+1. From the dropdown under the search bar, choose `Page path and screen class`.
+
+
+
```
Comment (Sboonny) [inline docs/authors-analytics-manual.md:19]:
```suggestion
```

File: docs/authors-analytics-manual.md [hunk 12]
```diff
@@ -0,0 +1,29 @@
+# Authors Analytics Manual
+
+If you are an author with access to the publication's Google Analytics Property (News), you can use this guide to view your articles engagement and search for articles by publication language."
+
+### Search by Language
```
Comment (Sboonny) [inline docs/authors-analytics-manual.md:5]:
```suggestion
## Search by Language
```

**Review by Sboonny — COMMENTED** [review]
Typo and wrong header syntax.

File: docs/authors-analytics-manual.md [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+1. From the top dropdown menu, select `News`.
+1. From the sidebar, click on `Reports`.
+1. From the secondary sidebar, select `Engagement`.
+1. Click on `Pages and Screens`.
+1. In the search bar, type the subpath for the desired language.
+1. From the dropdown under the search bar, choose `Page path and screen class`.
+
+
+
+### Filter Reports by Author
```
Comment (ahmadabdolsaheb) [inline docs/authors-analytics-manual.md:20]:
```suggestion
## Filter Reports by Author
```

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Pull request merged**